drivers/led/lp50xx.c

```diff
@@ -268,6 +268,45 @@ static int lp50xx_enable(const struct device *dev, bool enable)
 				      value);
 }
 
+static int lp50xx_turn_on(const struct device *dev)
+{
+	int err;
+
+	/* Enable hardware */
+	err = lp50xx_hw_enable(dev, true);
+	if (err < 0) {
+		LOG_ERR("%s: failed to enable hardware", dev->name);
+		return err;
+	}
+
+	/* Reset device */
+	err = lp50xx_reset(dev);
+	if (err < 0) {
+		LOG_ERR("%s: failed to reset", dev->name);
+		return err;
+	}
+
+	return 0;
+}
+
+static int lp50xx_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	switch (action) {
+	case PM_DEVICE_ACTION_SUSPEND:
+		return lp50xx_enable(dev, false);
+	case PM_DEVICE_ACTION_RESUME:
+		return lp50xx_enable(dev, true);
+	case PM_DEVICE_ACTION_TURN_ON:
+		return lp50xx_turn_on(dev);
+	case PM_DEVICE_ACTION_TURN_OFF:
+		return 0;
```

Comment:
Still not familiar with the PM API, but this has been previously rejected in the original PR #59852. Maybe there have been changes that I missed.

I just have one question : Why no corresponding `lp50xx_turn_off()`  to call `lp50xx_hw_enable(dev, false)` ?

Otherwise, LGTM.
Only caveat, configuration will be lost after `TURN_ON` from UVLO. Since there is no backup to restore, it is the responsibility of the application to restore the behavior before shutdown.